.github/workflows/publish-source-resolver.yml

```diff
@@ -11,6 +11,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          registry-url: "https://registry.npmjs.org"
+          node-version: 18.15
```

Comment:
Why is this necessary here but not on the other workflows?

Reply:
Can you open an issue to standardise this?

Reply:
https://github.com/orgs/noir-lang/projects/1/views/20?sliceBy%5Bvalue%5D=Week+40&pane=issue&itemId=40475271